powder_reduction.py

```diff
@@ -1,37 +1,142 @@
+#!/usr/bin/env python
+# coding: utf-8
+import numpy as np
 import scipp as sc
 from scipp import Dim
+from scipp.neutron.diffraction import load_calibration
+import smooth_data
 
+LAMBDA_RANGE = (0.7, 10.35)
```

Comment:
Make this a function parameter (with this as the default)? Hard-coded magic values are not optimal.

Reply:
Taken into account in the new version of the script

Reply:
I did not mean a function but a function parameter to the `powder_reduction` function.

Reply:
OK but how are you going to proceed for the range of d-spacing, which is different for each panel?

Reply:
This is about the lambda range, not d-spacing? I am not sure I understand your question.

Reply:
I agree that your question is related to the range of wavelength. But I do not understand why you would add the range of wavelengths in the arguments of `powder reduction` and not those of d-spacing (which is group-dependent).

Reply:
Yes I would also add the latter. You could just use `scipp.Variable` to provide the group-dependent bin edges if you like. But the script under discussion does not even use group-dependent edges, so if you keep it like that you can also do without a `Variable`.